Kickstarter-iOS/Features/PledgedProjectsOverview/CardView/PPOAddressSummary.swift

```diff
@@ -0,0 +1,46 @@
+import Foundation
+import SwiftUI
+
+struct PPOAddressSummary: View {
+  let address: String
+
+  var body: some View {
+    HStack(alignment: .firstTextBaseline) {
+      // TODO: Localize
+      Text("Shipping address")
+        .lineLimit(nil)
+        .font(Font(Constants.labelFont))
+        .foregroundStyle(Color(Constants.labelColor))
+        .frame(width: Constants.labelWidth, alignment: .leading)
+        .multilineTextAlignment(.leading)
+
+      Text(self.address)
```

Comment:
```suggestion
      Text(address)
```

Reply:
Our style rules currently require use of `self`.

Reply:
Okay. Thanks for letting me know. Looks good then